src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/LanguageServerWorkspaceFactory.cs

```diff
@@ -54,7 +54,11 @@ public LanguageServerWorkspaceFactory(
     public async Task InitializeSolutionLevelAnalyzersAsync(ImmutableArray<string> analyzerPaths)
     {
         var references = new List<AnalyzerFileReference>();
-        var analyzerLoader = Workspace.Services.GetRequiredService<IAnalyzerAssemblyLoaderProvider>().GetShadowCopyLoader();
+        var loaderProvider = Workspace.Services.GetRequiredService<IAnalyzerAssemblyLoaderProvider>();
+
+        // Load all analyzers into the default load context.  In the future, if we want to support reloading of analyzer
+        // references, we should instead load these into a dedicated ALC (using an IsolatedAnalyzerReferenceSet) here.
+        var analyzerLoader = loaderProvider.GetShadowCopyLoader(getSharedLoader: true);
```

Comment:
@dibarbet just a heads up on this.  i'll also have an in-person convo on what we can do here.